### DISCUSSION TITLE
Add method to suppress warnings about incompatibility with other mods

### DISCUSSION THREAD

**Discussion by mastery3** [iPortalTeam/ImmersivePortalsMod#1225] [Ideas]
I am using an older version of both Immersive Portals and Collective and I would like to hide the recent warning shown by Immersive Portals about Collective as Immersive Portals 2.2.3 appears to be compatible with Collective 5.15.

**Comment by qouteall** [comment]
Do you experience the issue that entities sometimes disappear when portal is near? If not I will remove the warning

**Comment by mastery3** [comment]
I cannot verify if the bug exists as the only nearby trackable entity is an armor stand in my storage room, and this entity has not disappeared. Xaero's World Map claims I am 390.59m away from the nearest village, which is approximately simulation distance. **I am on neither the latest version of Minecraft, Immersive Portals, or Collective.** I have noticed a zombie piglin _spawning_ once that couldn't have possibly gotten inside the surrounding structure, but this is likely to be an unrelated bug.

This is why I suggested being able to suppress the warning (a button to stop sending messages to chat about the incompatibility, as in the 'Many mods exist' warning) ourselves instead of asking the developer to do it, as there are many issues with the current approach:
- Some users are not on the latest versions of the incompatible mods, and the versions of the mods they are using are not incompatible with each other
- Some users may know the bug exists and do not care about the issue because working around it may be easier than removing this mod or the incompatible mod.
- Modpacks that use this mod may get this warning, in which case the user that is receiving the warning is incapable of fixing it. Also applies to servers

**Comment by mastery3** [comment]
Deleting Collective appears to not cause any dependency issues for my Minecraft instance. However, I still believe that a config option should be added to disable warnings for the reasons stated above.
